lib/reply.js

```diff
@@ -662,6 +669,13 @@ function serialize (context, data, statusCode) {
   return JSON.stringify(data)
 }
 
+/**
+ *
```

Comment:
In particular on this function, the code is difficult to follow if you do not have any context around why it is invoked and what the function is attempting to achieve. So a description in the docblock would be very helpful on this one.